tests/unit/plugins/modules/test_lvol.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2020- IBM, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import unittest
+from unittest import mock
+import re
+import copy
+
+from ansible_collections.ibm.power_aix.plugins.modules import lvol
+
+from .common.utils import (
+    AnsibleFailJson, fail_json, rootdir
+)
+
+
+params = {
+    "lv": "testlv",
+    "lv_new_name": None,
+    "vg": "testvg",
+    "lv_type": "jfs2",
+    "strip_size": None,
+    "extra_opts": None,
+    "num_of_logical_partitions": 5,
+    "copies": 1,
+    "policy": "maximum",
+    "pv_list": None
+}
+
+class TestCreateModifyLV(unittest.TestCase):
+    def setUp(self):
+        global params
+        self.module = mock.Mock()
+        self.module.params = params
+        rc, stdout, stderr = 0, "sample stdout", "sample stderr"
+        self.module.run_command.return_value = (rc, stdout, stderr)
+        self.module.fail_json = fail_json
+        self.lv_exist_path = rootdir + "lvol.lv_exists"
+
+
+    def _prepare_create_lv(self, fail=False):
+        # initialize global variable in lvol module
+        lvol.result = {
+            "changed": False,
+            "msg": '',
+            "cmd": '',
+            "stdout": '',
+            "stderr": ''
+        }
+
+        # compose the command that is ran in lvol module
+        self.cmd = "mklv -t %s -y %s -c %s -e %s %s %s %s %s %s" % (
+            self.module.params["lv_type"], 
+            self.module.params["lv"],
+            self.module.params["copies"], 
+            "x" if self.module.params["policy"] == "maximum" else "m", 
+            self.module.params["extra_opts"], 
+            "" if self.module.params["strip_size"] is None else \
+                "-S " + self.module.params["strip_size"], 
+            self.module.params["vg"], 
+            self.module.params["num_of_logical_partitions"],
+            "" if self.module.params["pv_list"] is None else \
+                " ".join(self.module.params["pv_list"])
+        )
+        print("I am here")
```

Comment:
Maybe a left over from the debug commands?

Reply:
ah yes, that is my debug prints